### PR TITLE
fix missing key virustotal.py

### PR DESCRIPTION
**Pull request by CheYenBzh**
Hi,

Here is a fix for the 'missing key "submission_names"' that happens sometime.

Here are the related issues:
https://github.com/MISP/misp-modules/issues/50
https://github.com/MISP/MISP/issues/1466

CheYen

### CODE REVIEW

File: misp_modules/modules/expansion/virustotal.py [hunk 1]
```diff
@@ -136,12 +136,20 @@ def getMoreInfo(req, key):
                             params = {"hash":hsh, "apikey":key})
       
       malsample = sample.content
-      r.append({"types":["malware-sample"], 
-                "categories":["Payload delivery"],
-                "values":data["submission_names"],
-                "data": str(base64.b64encode(malsample), 'utf-8')
-                }
-              )  
+      if isset(data, "submission_names"):
```
Comment (FloatingGhost) [inline misp_modules/modules/expansion/virustotal.py]: isset is not valid python

Try 

```python
if "submission_names" in data:
  ...
```

File: misp_modules/modules/expansion/virustotal.py [hunk 2]
```diff
@@ -136,12 +136,20 @@ def getMoreInfo(req, key):
...
+        r.append({"types":["malware-sample"], 
+                  "categories":["Payload delivery"],
+                  "values":data["submission_names"],
+                  "data": str(base64.b64encode(malsample), 'utf-8')
+                  }
+                )
+      else:
+        r.append({"types":["malware-sample"], 
+                  "categories":["Payload delivery"],
+                  "values":data["submission_names"],
```
Comment (FloatingGhost) [inline misp_modules/modules/expansion/virustotal.py]: You've changed nothing - you're still trying to add the key submission_names which you have determined to not exist.

File: misp_modules/modules/expansion/virustotal.py (hunk 1 above)
Comment (FloatingGhost) [inline misp_modules/modules/expansion/virustotal.py]: I retract this. Whoever built the module put it in.

Comment (FloatingGhost) [inline misp_modules/modules/expansion/virustotal.py]: oh it was me. Haha.

**Comment by Rafiot** [comment]
Done in https://github.com/MISP/misp-modules/pull/89

**Pull request closed without merging**